### ISSUE TITLE
Issue with Refined Storage pattens not accepting Silicon from Emendatus Engimatica

### ISSUE THREAD

**Issue by Zoidey** [NillerMedDild/Enigmatica6#1468]
Trying to create a Refined Storage Pattern to auto craft Raw Advanced Processors but it will only accept its own Silicon (Refined Storage) and not the Emendatus Engimatica version of the Silicon. 

I have tried selecting "Exact" in the pattern with the Emendatus Engimatica Silicon and Normal. Both don't work as it won't accept anything other than the silicon from Refined Storage.

There are a few recipes that have this issue that I've heard from my friends that play on my server, not sure what exact recipes other than the one I've mentioned.

**Comment by NillerMedDild** [comment]
Which modpack version are you on? The template is there for a reason, please follow it :)

**Comment by Zoidey** [comment]
Ah yes, sorry about that. The version I'm using for now is 0.4.0, will upgrade to the newest one shortly.

**Comment by Darkere** [comment]
I feel like this might be specific to your server. Pretty sure we would have heard about this otherwise.

That said I have no idea what could cause this. Maybe the restart will fix it.

**Comment by Zoidey** [comment]
Restart hasn't fixed it as the server restarts automatically every 6 hours. I've only had this issue since updating to version 0.4.0. I'll check again after updating to the newer version as that might fix it.

It could be my server but I haven't changed anything other than the config for FTB Backups.

**Comment by NillerMedDild** [comment]
Do you see errors in chat when you log on? May we have your latest.log please?

**Comment by Zoidey** [comment]
As far as I could or couldn't see, there were no errors coming up.

It seems to be working now, I just had to manually recreate the pattern several times for the pattern to accept the use of Emendatus Engimatica Silicon and not the Refined Storage Silicon. 

I have noticed that if you use the auto-create pattern it'll always use the non-Emendatus Engimatica items (ingots etc) and anything with glass in the pattern adds Connectected glass to the pattern by default rather than the Minecraft standard glass.

Updating to version 0.4.1 might have helped. I'll give it another day or so and see if there are any more issues.
